src/platform/plugins/shared/workflows_management/public/features/run_workflow/ui/workflow_execute_event_form.tsx

```diff
@@ -41,41 +39,25 @@ interface Alert {
   };
 }
 
+interface AlertSelection {
+  _id: string;
+  _index: string;
+}
+
+interface AlertTriggerInput {
+  event: {
+    alertIds: AlertSelection[];
+    triggerType: 'alert';
+  };
+}
+
```

Comment:
this is duplicated in src/platform/plugins/shared/workflows_management/server/workflows_management/utils/preprocess_alert_inputs.ts, export it?